interfaces/Portalicious/src/app/pages/project-payments/components/payment-summary-card/payment-summary-card.component.html

```diff
@@ -0,0 +1,80 @@
+<p-card
+  styleClass="[&_.p-card-content]:flex-end group relative h-full border border-transparent *:h-full focus-within:border-purple-500 hover:border-purple-500 [&_.p-card-title]:flex-grow"
+>
+  <ng-template pTemplate="title">
+    <a
+      [routerLink]="paymentLink(projectId(), paymentId())"
+      class="after:absolute after:inset-0 after:block group-focus-within:text-purple group-hover:text-purple"
+    >
+      @if (!paymentId()) {
+        <app-skeleton-inline />
+      } @else {
+        <span
+          class="line-clamp-2 text-ellipsis"
+          [title]="
+            'Payment ' + (paymentDate() | date: 'dd-MM-yyyy, hh:mm')
+              | translatableString
+          "
+        >
+          {{
+            'Payment ' + (paymentDate() | date: 'dd-MM-yyyy, hh:mm')
+              | translatableString
```

Comment:
```suggestion
```

I don't think `translatableString` is doing what you think it is doing here. If you want this to be translated, and therefore end up in the `xlf` files, you need to use `$localize` in the ts, or the `i18n` attribute in the template.

This is a tricky use case though. If you're struggling with it send me a DM.